### DISCUSSION TITLE
DuckDuckGo

### DISCUSSION THREAD

**Discussion by OfficialCRUGG** [catppuccin/catppuccin#2145] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by nekowinston** [comment]
Please see https://github.com/catppuccin/duckduckgo

**Reply by OfficialCRUGG** [reply]
I see, thanks. Expected it to be a userstyle so I didn't see it had its own thing. Sorry.

**Reply by nekowinston** [reply]
Yeah, there has been some work on https://github.com/catppuccin/duckduckgo/pull/10, but it appears to be stalled.
